client/webserver/site/src/js/wallets.ts

```diff
@@ -233,8 +233,13 @@ export default class WalletsPage extends BasePage {
     Doc.hide(this.page.forms)
   }
 
-  copyAddress () {
-    navigator.clipboard.writeText(this.page.depositAddress.textContent || '')
+  async copyAddress () {
+    const page = this.page
+    navigator.clipboard.writeText(page.depositAddress.textContent || '')
+    Doc.show(page.copyAlert)
```

Comment:
We should only show the "Copied!" alert if `writeText` says it succeeded.

```js
navigator.clipboard.writeText(textContent).then(() => {
	Doc.show(page.copyAlert)
	...
}, (reason) => {
	console.error('Unable to copy:', reason) // or a different alert, whatever
})
```

https://github.com/decred/dcrdata/commit/c9521242ea01902ed3452250306d67ac54932824